db/schema.rb

```diff
@@ -191,7 +191,7 @@
   end
 
   create_table "nebraska_dashboard_cases", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
-    t.datetime "month", default: "2022-05-10 19:40:48", null: false
+    t.datetime "month", default: "2022-05-11 19:17:16", null: false
```

Comment:
None of this should be committed, also same with pie_erd (I can't comment on that file because it's binary)